common/src/lib.rs

```diff
@@ -398,38 +360,6 @@ pub fn wait_key(prog_id: H256, msg_id: H256) -> Vec<u8> {
     key
 }
 
-pub fn get_code(code_hash: H256) -> Option<Code> {
-    sp_io::storage::get(&code_key(code_hash, CodeKeyPrefixKind::Code))
-        .map(|bytes| Code::decode(&mut &bytes[..]).expect("Code encoded correctly; qed"))
-}
-
-pub fn set_code(code_hash: H256, code: &Code) {
-    sp_io::storage::set(
-        &code_key(code_hash, CodeKeyPrefixKind::Code),
-        &code.encode(),
-    )
-}
-
-pub fn get_original_code(code_hash: H256) -> Option<Vec<u8>> {
-    sp_io::storage::get(&code_key(code_hash, CodeKeyPrefixKind::OriginalCode))
-}
-
-pub fn set_original_code(code_hash: H256, code: &[u8]) {
-    sp_io::storage::set(&code_key(code_hash, CodeKeyPrefixKind::OriginalCode), code)
-}
-
-pub fn set_code_metadata(code_hash: H256, metadata: CodeMetadata) {
-    sp_io::storage::set(
-        &code_key(code_hash, CodeKeyPrefixKind::CodeMetadata),
-        &metadata.encode(),
-    )
-}
-
-pub fn get_code_metadata(code_hash: H256) -> Option<CodeMetadata> {
-    sp_io::storage::get(&code_key(code_hash, CodeKeyPrefixKind::CodeMetadata))
-        .map(|data| CodeMetadata::decode(&mut &data[..]).expect("data encoded correctly"))
-}
-
 pub fn set_program_initialized(id: H256) {
```

Comment:
should these-like methods be also inside the gear-program pallet?

Reply:
In #788 . This PR deals with code storage only.